examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-base/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[cmp-examples.base]"
-    embed="[cmp-examples.fonts,core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,cmp-examples.fonts,core.wcm.components.pdfviewer.v1,core.wcm.components.image.v3,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,cmp-theme.core-components-clean,core.wcm.components.form.container.v2,cmp-examples.site]"/>
+    embed="[cmp-examples.fonts,core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,cmp-examples.fonts,core.wcm.components.pdfviewer.v1,core.wcm.components.image.v3,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,cmp-theme.core-components-clean,core.wcm.components.form.container.v2,core.wcm.components.embed.v1,cmp-examples.site]"/>
```

Comment:
Core component examples uses v2 of embed component, but the fix hasn't been applied there.